hermes/probe/read/read_file.go

```diff
@@ -0,0 +1,80 @@
+// TODO (#70) add license header and author line
+// TODO (#69) add doc strings
+
+package read
+
+import (
+	"context"
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/cloudprober/logger"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	m "github.com/googleinterns/step224-2020/hermes/probe/metrics"
+)
+
+func checkSumFromName(fileName string) (string, error) {
+	if len(fileName) < 11 {
+		return "", fmt.Errorf("The filename stored in the journal is shorter than 11 characters, hence does not match Hermes_id_checksum")
+	}
+	return fileName[10:len(fileName)], nil
+}
+
+func ReadFile(ctx context.Context, target *probe.Target, fileID int32, client stiface.Client, logger *logger.Logger) error {
+	bucketName := target.Target.GetBucketName()
+	fileName, ok := target.Journal.Filenames[fileID]
+	if !ok {
+		var status m.ExitStatus
+		status = m.FileMissing
+		return fmt.Errorf("ReadFile(ID: %d).%v could not read file as the file with the provided ID does not exist", fileID, status)
+	}
+	prefix := fileName[0:9]
+	query := &storage.Query{Prefix: prefix}
```

Comment:
Just define prefix inline.

Reply:
changed to how it was handled in create.go